annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -572,20 +595,56 @@ private void validateTag(Annotation webAnnotation) throws ParamValidationExcepti
      */
     private void validateTranscription(Annotation webAnnotation)
 	    throws ParamValidationException, RequestBodyValidationException, PropertyValidationException {
-	validateTranscriptionWithFullTextResource(webAnnotation.getBody());
+		validateTranscriptionWithFullTextResource(webAnnotation.getBody());
+
+    	// "source" becomes mandatory as soon as you have a "scope" in the target
+	    if (webAnnotation.getTarget() != null && !StringUtils.isBlank(webAnnotation.getTarget().getScope()) && StringUtils.isBlank(webAnnotation.getTarget().getSource()))
+	        throw new PropertyValidationException(ParamValidationException.MESSAGE_MISSING_MANDATORY_FIELD,
+		        I18nConstants.MESSAGE_MISSING_MANDATORY_FIELD, new String[] { "transcription.target.scope" });
     }
 
-    /**
-     * This method validate entity body
+	/**
+     * This method validates entity body.
      * 
      * @param body The entity body
      * @throws ParamValidationException
+	 * @throws PropertyValidationException 
      */
-    private void validateAgentBody(Body body) throws ParamValidationException {
-	if (!(body instanceof EdmAgentBody)) {
+    private void validateAgentBody(Body body) throws ParamValidationException, PropertyValidationException {
+		if (!(body instanceof EdmAgentBody)) {
```

Comment:
The check must not be based on the class, but on the body.getType, see validation of body type for transcriptions

Reply:
Replaced similar to transcriptions